library/std/src/os/unix/net/ancillary.rs

```diff
@@ -21,6 +21,27 @@ mod libc {
     pub type uid_t = u32;
 }
 
+#[cfg(freebsd13)]
+#[unstable(feature = "sockcred2", issue = "none")]
+#[derive(Clone)]
+pub struct sockcred2 {
```

Comment:
This structure is already defined in libc.  Why duplicate it here?

Reply:
When I started it last year libc version here was not updated but I forgot to remove these afterwards indeed :)